### PR TITLE
Dubbo server config use virtualHost first

### PR DESCRIPTION
**Pull request by alaneuler**

### CODE REVIEW

File: bootstrap/bootstrap-dubbo/src/main/java/com/alipay/sofa/rpc/bootstrap/dubbo/DubboProviderBootstrap.java [hunk 1]
```diff
@@ -116,7 +116,11 @@ private void copyServers(ProviderConfig<T> providerConfig, ServiceConfig service
     private void copyServerFields(ServerConfig serverConfig, ProtocolConfig protocolConfig) {
         protocolConfig.setId(serverConfig.getId());
         protocolConfig.setName(serverConfig.getProtocol());
-        protocolConfig.setHost(serverConfig.getHost());
+        String host = serverConfig.getVirtualHost();
+        if (host == null) {
```
Comment (OrezzerO) [inline bootstrap/bootstrap-dubbo/src/main/java/com/alipay/sofa/rpc/bootstrap/dubbo/DubboProviderBootstrap.java:120]: how about  host is empty？

**Review by OrezzerO — APPROVED** [review]
add a test Case

**Review by OrezzerO — CHANGES_REQUESTED** [review]
need unit test

File: bootstrap/bootstrap-dubbo/src/main/java/com/alipay/sofa/rpc/bootstrap/dubbo/DubboProviderBootstrap.java (hunk 1 above)
Comment (NeGnail) [inline bootstrap/bootstrap-dubbo/src/main/java/com/alipay/sofa/rpc/bootstrap/dubbo/DubboProviderBootstrap.java:120]: 可以用下StringUtils.isBlank

**Pull request closed without merging**